code/datums/components/pricetag.dm

```diff
@@ -1,13 +1,16 @@
 /datum/component/pricetag
-	var/datum/bank_account/owner
-	var/profit_ratio = 1
+	var/default_profit_ratio = 1
+	var/list/payees = list()
```

Comment:
Could you put an autodoc string `///` above this explaining what it's likely to contain